src/SPC/doctor/item/LinuxToolCheckList.php

```diff
@@ -38,6 +40,7 @@ class LinuxToolCheckList
         'tar', 'unzip', 'gzip', 'gcc',
         'bzip2', 'cmake', 'patch', 'which',
         'xz', 'libtool', 'gettext-devel',
+        'perl', 'patchelf',
```

Comment:
I ran into this error every single time I set up a RHEL installation. perl is needed for perl-FindBin on rhel, which a lot of configure scripts from libraries need